tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -3967,6 +3968,31 @@ TEST_F(CompilerTest, PreprocessWithDebugOptsThenOk) {
     "int BAR;\n", text.c_str());
 }
 
+// Make sure that '#line 1 "<built-in>"' won't blow up when preprocessing.
+TEST_F(CompilerTest, PreprocessCheckBuiltinIsOk) {
+  CComPtr<IDxcCompiler> pCompiler;
+  CComPtr<IDxcOperationResult> pResult;
+  CComPtr<IDxcBlobEncoding> pSource;
+  
+  VERIFY_SUCCEEDED(CreateCompiler(&pCompiler));
+  CreateBlobFromText(
+    " \r\n\t \r\n\r\n"
```

Comment:
nit: The blank line here probably isn’t needed and you should be able to remove the `\r` characters.